app/controllers/callbacks_controller.rb

```diff
@@ -0,0 +1,14 @@
+class CallbacksController < Devise::OmniauthCallbacksController
```

Comment:
@stedmanblake, can we call this `OmniauthCallbacksController` to match what we're inheriting from?